SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/infraspecdev/).
+
+If you find any security vulnerabilities in our open source projects, please report them. We will ensure that your findings are passed along to the appropriate maintainers for remediation.
+
+## Reporting Security Issues
+
+If you believe you have found a security vulnerability in any GitHub-owned repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+Instead, please send an email to <security@infrapsec.dev>.
+
+Please include as much of the information listed below as you can to help us better understand and resolve the issue:
+
+* The type of issue (e.g., improper input handling, token exposure, unauthorized access to GitHub API, or API abuse in the spell and grammar checking process)
```

Comment:
Change examples to remove GitHub examples like unauthorized access to GitHub API.

Reply:
Done b473732540c72f0015f987b2ab84b7fc401a831c